<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QRegisterSupplyInfo</class>
 <widget class="BaseDlg" name="QRegisterSupplyInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QRegisterSupplyInfo</string>
  </property>
  <widget class="QPushButton" name="ok_btn">
   <property name="geometry">
    <rect>
     <x>357</x>
     <y>382</y>
     <width>116</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
   <property name="buttons" stdset="0">
    <string>btn_dlg</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_btn">
   <property name="geometry">
    <rect>
     <x>487</x>
     <y>382</y>
     <width>116</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
   <property name="buttons" stdset="0">
    <string>btn_dlg</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridWidget" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>550</width>
     <height>276</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="horizontalSpacing">
     <number>0</number>
    </property>
    <property name="verticalSpacing">
     <number>10</number>
    </property>
    <item row="4" column="2">
     <widget class="QLineEdit" name="reagent_sn">
      <property name="minimumSize">
       <size>
        <width>240</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>5</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="0">
     <widget class="QLineEdit" name="reagent_lot">
      <property name="minimumSize">
       <size>
        <width>240</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QComboBox" name="ReagentName_comboBox">
      <property name="minimumSize">
       <size>
        <width>240</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>瓶号</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QDateEdit" name="reagent_date">
      <property name="minimumSize">
       <size>
        <width>240</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>40</height>
       </size>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="displayFormat">
       <string>yyyy-MM-dd</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="reagent_pos">
      <property name="minimumSize">
       <size>
        <width>240</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>试剂位</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>失效日期</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>试剂批号</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="0">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>5</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>名称</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BaseDlg</class>
   <extends>QWidget</extends>
   <header>shared/basedlg.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
